docs/pages/guides/authentication.md

```diff
@@ -420,6 +420,77 @@ export default function App() {
 
 </Tab>
 
+<Tab>
+
+Auth code responses (`ResponseType.Code`) will only work in native with `useProxy: true` or `usePKCE: true`. The PCKE OAuth workflow returns an authentication code which can be exchanged for a short-lived access token and a long-lived refresh-token. This allows for long-session Dropbox connections.
+
+<SnackInline label='Dropbox Auth Code' dependencies={['expo-auth-session', 'expo-web-browser']}>
+
+<!-- prettier-ignore -->
+```tsx
+import * as React from 'react';
+import * as WebBrowser from 'expo-web-browser';
+import { makeRedirectUri, useAuthRequest } from 'expo-auth-session';
+import { Button, Platform } from 'react-native';
+
+/* @info <strong>Web only:</strong> This method should be invoked on the page that the auth popup gets redirected to on web, it'll ensure that authentication is completed properly. On native this does nothing. */
+WebBrowser.maybeCompleteAuthSession();
+/* @end */
+
+// Endpoint
+const discovery = {
+  authorizationEndpoint: 'https://www.dropbox.com/oauth2/authorize',
+  tokenEndpoint: 'https://www.dropbox.com/oauth2/token',
+};
+
+export default function App() {
+  /* @info request will containt the <b>codeVerifier</b> which will need to be used with the Dropbox token endpoint to exchange the code for an access token and a refresh token*/
+  const [request, response, promptAsync] = useAuthRequest(
+    {
+      clientId: 'CLIENT_ID',
+      // There are no scopes so just pass an empty array
+      scopes: [],
+      usePKCE: true,
+      redirectUri: makeRedirectUri({
+        // For usage in bare and standalone
+        native: 'your.app://redirect',
+      }),
+      //Needed to obtain refresh token
```

Comment:
lint the code

Reply:
@EvanBacon - how do I proceed to do this? 🤔

Reply:
@EvanBacon - any hint please?

Reply:
it looks like this block has prettier-ignore due to the `@info` annotations, you can try removing that and running prettier here then restoring the `@info` annotation formatting. a red flag here is `//Needed to` -- there should be a space after `//`. i suspect there may be other style concerns also